lib/serviceCatalogue.js

```diff
@@ -41,7 +51,7 @@ const getServices = (apiConfig, path) => {
           reject(err);
         }
       });
-    }).on('error', err => reject(err));
+    }).on('error', (err) => reject(errMsg(apiConfig.host)));
```

Comment:
It might be good to log it to a file for us to help people debug but that feels like a future improvement. Not for this PR